tasks/install-helm.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+- name: Set os_arch variable
+  set_fact:
+    os_arch: "{{ ansible_system | lower }}-amd64"
```

Comment:
Even though the Helm installation shouldn't be included in the first place, as an additional side note: The `-amd64` assumption will already introduce limitations here.

Reply:
I'm aware of this but for the sake of simplicity I decided to support only x64 since this will be only used now

Reply:
Btw the same limitations come with Docker